src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/ConcurrencyLimiter.cs

```diff
@@ -38,7 +38,23 @@ public sealed class ConcurrencyLimiter : RateLimiter
         /// <param name="options">Options to specify the behavior of the <see cref="ConcurrencyLimiter"/>.</param>
         public ConcurrencyLimiter(ConcurrencyLimiterOptions options)
         {
-            _options = options ?? throw new ArgumentNullException(nameof(options));
+            ArgumentNullException.ThrowIfNull(options);
```

Comment:
This wont work, it's a new API starting in 6.0 and we target netstandard2.0.
You should take a look at code spaces if you don't want to build locally.